hapi/hedera-protobufs/platform/tss/tss_message_map_key.proto

```diff
@@ -10,11 +10,9 @@
  */
 syntax = "proto3";
 
-package com.hedera.hapi.node.state.tss;
+package com.hedera.hapi.platform.tss;
```

Comment:
Does this still belong to "platform" ? Why can't this go to com.hedera.hapi.services.auxiliary.tss ? Also, are the models defined in this file even used anywhere? If not, then perhaps we should delete this file.

Reply:
Resolved by moving this class to a new directory